composer.json

```
@@ -13,7 +13,8 @@
         "symbiote/silverstripe-gridfieldextensions": "^3.1"
     },
     "require-dev": {
-        "phpunit/phpunit": "^5.7",
+        "sminnee/phpunit": "^5.7.29",
+        "sminnee/phpunit-mock-objects": "^3.4.9",
```

`sminnee/phpunit` now requires `sminnee/phpunit-mock-objects`, so you don't need that line anymore.

```suggestion
        "sminnee/phpunit": "^5.7.29"
```